packages/hub/src/scope.ts

```diff
@@ -155,6 +162,13 @@ export class Scope implements ScopeInterface {
     return this;
   }
 
+  /**
+   * @inheritdoc
+   */
+  public getExtras(): Extras {
```

Comment:
Hmm `Scope` was originally designed to be write-only. We've deviated from that with tracing, but I'm uneasy to add more API surface because of sessions